src/apps/bristleback_apps/nortek/user_code/user_code.cpp

```diff
@@ -335,11 +335,11 @@ void loop(void) {
         "DATA, %" PRIu64 ", %s, %u, %u, %u, %u, %u, %u, %u, %u, %.4f, %.4f, %.4f, %u, %u, %u, %.4f, %.4f, %.4f, %.4f, %.4f, %.4f, %.4f, %u, %u, %.4f, %.4f\n",
         this_uptime, rtcTimeBuffer,nortekData.month, nortekData.day, nortekData.year, nortekData.hour, nortekData.minute, nortekData.second, nortekData.error_code, nortekData.status_code, nortekData.v_b1_x, nortekData.v_b2_y, nortekData.v_b3_z, nortekData.amp_b1_x, nortekData.amp_b2_y, nortekData.amp_b3_z, nortekData.batt_v, nortekData.sound_speed, nortekData.heading, nortekData.pitch, nortekData.roll, nortekData.pressure, nortekData.temperature, nortekData.an_in_1, nortekData.an_in_2, nortekData.speed, nortekData.direction
       );
-      bm_printf(0,
+      bm_printf(0, 
```

Comment:
nit: remove the trailing whitespaces. I use the automatic feature in vscode. I believe clang-format should also do this. 

Some context on clang-fomrat. Some but not all of the files are clang-formatted. We are slowly trying to transition to having them all be clang-formatted. For files where they aren't formatted yet we useually try to seperate the clang-format changes from the actual code changes to make it easier to review. Sometimes that is just clang-formating the next commit after review before merging, sometimes that is a separate PR with only clang-format changes. Other times it may be that you clang-format it and then just add a comment in the PR saying "here is the change! the rest is clang-format". Generally up to the PR creator's discretion. @towynlin any other thoughts on clang-format?

TBH I forget to clang-format sometimes so this is a good reminder for me too! 😅

Reply:
Well said! 👍

Reply:
Ahh shoot, I saw this too late apologies.